Tests/DatadogTests/Datadog/Core/Persistence/Writing/FileWriterTests.swift

```diff
@@ -19,6 +19,42 @@ class FileWriterTests: XCTestCase {
         super.tearDown()
     }
 
+    func testItWritesDataWithMetadataToSingleFileInTLVFormat() throws {
+        let writer = FileWriter(
+            orchestrator: FilesOrchestrator(
+                directory: temporaryDirectory,
+                performance: PerformancePreset.mockAny(),
+                dateProvider: SystemDateProvider()
+            ),
+            encryption: nil,
```

Comment:
suggestion/ How about adding one more test that makes sure metadata blocks are encrypted as well?

Reply:
Good call, added.